lib/index.js

```diff
@@ -25,13 +25,13 @@ export default class WdioImageComparisonService extends BaseClass {
         /**
          * Saves an image of an element
          */
-        browser.addCommand('saveElement', function (element, tag, saveElementOptions = {}) {
+        browser.addCommand('saveElement', function (element, tag, saveElementOptions = {}, folderOptions = {}) {
```

Comment:
Hmm, 

I'm not a big fan of adding more options to the methods and secondly this will not work if `save*Options` are note and `folderOptions` are provided.
What do you think about adding this to the `save*Options`-object?

Reply:
The changes should be in place now.